core/templates/pages/classroom-page/classroom-page.controller.ts

```diff
@@ -54,7 +54,12 @@ angular.module('oppia').directive('classroomPage', [
             ctrl.bannerImageFileUrl = UrlInterpolationService.getStaticImageUrl(
               '/splash/books.svg');
 
-            PageTitleService.setPageTitle(ctrl.classroomName + ' - Oppia');
+            ctrl.classroomDisplayName = (
+              ctrl.classroomName.charAt(0).toUpperCase() +
```

Comment:
Do we still need to bind `classroomName` in the ctrl?

Reply:
Oh, right. Changed!